tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -145,6 +150,44 @@ filings.forEach((filing: any, index: number) => {
       expect(wrapper.find('.hide-details').text()).toBe('Hide Documents')
     })
 
+    itIf(isHorizontalAmalgamation(filing))('horizontal amalgamation filing', async () => {
+      // verify data
+      expect(vm.getFilings.length).toBe(1) // sanity check
+      const item = vm.getFilings[0]
+      expect(item.businessIdentifier).toBe('BC1234567')
+      expect(item.data.amalgamationApplication.type).toBe(FilingSubTypes.AMALGAMATION_HORIZONTAL)
+      expect(item.displayName).toBe('Amalgamation Application - Horizontal')
+      expect(item.name).toBe(FilingTypes.AMALGAMATION_APPLICATION)
+      expect(item.status).toBe(FilingStatus.COMPLETED)
+
+      // verify display
+      expect(wrapper.find('.item-header-title').text()).toBe('Amalgamation Application - Horizontal')
```

Comment:
This is an old title (has been changed recently). Probably the displayName in the JSON file needs to be updated.

Reply:
I'm looking at main and I see the line is exactly the same Sev:
https://github.com/bcgov/business-filings-ui/blob/43b527cce8469b8df1a22f562799ae740f568cd7/tests/unit/FilingHistoryList2.spec.ts#L164

Do we want it to be something different?

Reply:
Yeah, these filings have new titles - see https://github.com/bcgov/lear/pull/2469.

Unfortunately, unit tests are happy with bad data if that's what you're comparing to 😆

Reply:
Much appreciated! In the end, it doesn't matter if this is fixed in main or LE f/b.